sw/otbn/code-snippets/p256_ecdsa_verify_test.s

```diff
@@ -57,6 +62,16 @@ ecdsa_verify_test:
 
 .data
 
+blinding_param:
+  .word 0x7ab203c3
+  .word 0xd6ee4951
+  .word 0xd5b89b43
+  .word 0x409d2b56
+  .word 0x8e9d2186
+  .word 0x1de0f8ec
+  .word 0x0fa0bf9a
+  .word 0xa21c2147
+
```

Comment:
This seems wrong, the verification routine shouldn't need a blinding param

Reply:
Good point, removed.